apps/api/src/Greencheck/Checker.php

```diff
@@ -62,7 +69,10 @@ public function __construct(
         $this->statsdClient = $statsdClient;
         $this->logger = $logger;
         $this->producer = $producer;
+        $this->redis = new Client();
```

Comment:
this will work if the default dsn is fine, but if we need to change it, it would be better to inject it instead of instantiating a new one here.